addons/medical_statemachine/functions/fnc_enteredStateCardiacArrest.sqf

```diff
@@ -22,13 +22,15 @@ if (isNull _unit || {!isNil {_unit getVariable QEGVAR(medical,causeOfDeath)}}) e
 };
 
 // 10% possible variance in cardiac arrest time
-private _time = GVAR(cardiacArrestTime);
-_time = _time + _time * random [-0.1, 0, 0.1];
+private _timeDiff = CBA_missionTime - (_unit getVariable [QGVAR(cardiacArrestTimeLastUpdate), 0]);
+if(_timeDiff > 2) then {
```

Comment:
this is added as it was noticed that if someone's SpO2 was at the fatal level, and someone started their heart through CPR, they would immediately fall back into CA and timer would restart